### PR TITLE
Exports MusicControl.Command

### LINKED ISSUE #358: TypeScript Error: MusicControl.on expecting enum
# Description

1. Sample code (provide repo url or sample code)
```
    MusicControl.on('play', () => {
      setIsPlaying(true);
    });
```

produces this error...

Screen Shot 2020-11-17 at 9 47 38 AM

The `Command` enum doesn't seem to be exported so there's no way to use the expected parameter. Strings seem to still work fine.

2. Platform ?

   - [x] iOS
   - [x] Android
   
   
3. Device
  - [x] Simulator
  - [x] Real device

### PR DESCRIPTION
**Pull request by redreceipt**
Exports Command enum for use in `MusicControl.on`

Fixes #358

### CODE REVIEW

**Comment by tanguyantoine** [comment]
Would you mind updating the readme accordingly? 🙏

**Comment by redreceipt** [comment]
like that?

File: README.md [hunk 1]
```diff
@@ -199,43 +199,43 @@ componentDidMount() {
```
Comment (tanguyantoine) [inline README.md:199]:
```suggestion
import { Command } from 'react-native-music-control'
```

File: README.md [hunk 2]
```diff
@@ -199,43 +199,43 @@ componentDidMount() {
 
     // on iOS this event will also be triggered by audio router change events
     // happening when headphones are unplugged or a bluetooth audio peripheral disconnects from the device
-    MusicControl.on('pause', ()=> {
+    MusicControl.on(MusicControl.Command.pause, ()=> {
```
Comment (tanguyantoine) [inline README.md:202]: After we import the `Command` enum, we can use it as follow
```suggestion
    MusicControl.on(Command.pause, ()=> {
```

**Review by tanguyantoine — COMMENTED** [review]
I dropped some suggestion and then it should be fine

**Comment by bradleyflood** [comment]
Bump @redreceipt thanks bro :)

**Comment by redreceipt** [comment]
I made the suggested changes

**Comment by bradleyflood** [comment]
Thanks bro @redreceipt 

@tanguyantoine You happy?

**Comment by tanguyantoine** [comment]
Almost, I'll update it once it has been merged. thank you @redreceipt

**Comment by tanguyantoine** [comment]
published under `1.3.0`

**Pull request merged**